src/pages/components/[libId]/[componentId].tsx

```diff
@@ -48,11 +47,22 @@ export const getServerSideProps: GetServerSideProps = async (ctx) => {
         readmePromise,
     ]);
 
+    // Generate meta description from static config
+    const componentMeta = await getComponentMeta(
```

Comment:
1) Let's change method params and make it sync:
```ts
const componentMeta = getComponentMeta({
    libId: lib.config.id,
    libTitle: lib.config.title
    componentId: component.id,
    componentTitle: component.title,
    t,
});
```

2) Move it from `getServerSideProps` to `ComponentPage`, where we already have the `t` method with the correct locale.

3) `fallbackDescription` parameters is not needed, it is already defined inside the method.

4) Need to add `component-meta` to `const i18nPropsPromise = getI18nProps(ctx, ['component', 'libraries-info', 'component-meta']);` above.